store/actions/index.js

```diff
@@ -0,0 +1,65 @@
+import getConfig from 'next/config'
+
+// To fetch .env variables from next.config.js
+const {publicRuntimeConfig} = getConfig()
+const {holidayApiURL, holidayApiKey} = publicRuntimeConfig
+
+
+import {
+  SET_COUNTRY,
+  SET_YEAR,
+  RECIEVE_COUNTRIES,
+  REQUEST_HOLIDAYS,
+  RECIEVE_HOLIDAYS
+} from './action_types'
+
+function receiveCountries(value) {
```

Comment:
Isn't it a standard to pass action.type and action.value ? That's why i set the parameter name as value.

Reply:
I think `type` something that redux has standardised from start instead of using `action_type`. But I am not aware about using `value` as a standard parameter on redux site. It seems they use explicit parameter names for it.

Some might be definitely following that practice but it is not really good practice in my opinion. Specially when more than one developers are working on it.